.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -54,3 +54,12 @@ python collect_env_details.py
 ### Additional context
 
 <!-- Add any other context about the problem here. -->
+
+
+
+### Learn more about our other projects:
+
+- [Flash](https://github.com/PyTorchLightning/lightning-flash): Collection of tasks for fast prototyping, baselining, finetuning and solving problems with deep learning
+- [Bolts](https://github.com/PyTorchLightning/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks and more for research and production with PyTorch Lightning and PyTorch
+- [Metrics](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.
+- [Lightning Transformers](https://github.com/PyTorchLightning/lightning-transformers): Flexible interface for high performance research using SOTA Transformers leveraging Pytorch Lightning, Transformers, and Hydra.
```

Comment:
I do not think it is the right place for promo

Reply:
Curious to hear the team thoughts there. I think the Lighting Ecosystem is worth mentioning, especially for new users.

Reply:
yes, but here you mentioning it for all users, not new users...
for new users we had this bot - https://github.com/PyTorchLightning/pytorch-lightning/blob/release/1.0.x/.github/workflows/greetings.yml